src/components/filesystem/filesystem.html

```diff
@@ -266,7 +266,7 @@
   <div v-if="progressMonitors.length>0" class="progressholder">
     <progressbar 
         v-for="progress in progressMonitors" v-if="progress.show"
-      v-on:hide-progress="progressMonitors.pop(progress)"
+      v-on:hide-progress="progressMonitors.splice(progressMonitors.indexOf(progress), 1)"
```

Comment:
great catch! Does this fix the random closing of upload progress when concurrently uploading many files?

Reply:
yes. Note also that uploads are no longer concurrent. This makes it possible to check available space after each upload and also should fix problems whereby the ui would get out of sync